packages/engine/Source/Scene/ModelComponents.js

```diff
@@ -589,7 +589,7 @@ function Primitive() {
    * The feature IDs associated with this primitive. Feature ID types may
    * be interleaved
    *
-   * @type {Array.<ModelComponents.FeatureIdAttribute|ModelComponents.FeatureIdImplicitRange|ModelComponents.FeatureIdTexture>}
+   * @type {ModelComponents.FeatureIdAttribute[]|ModelComponents.FeatureIdImplicitRange[]|ModelComponents.FeatureIdTexture[]}
```

Comment:
This new syntax doesn't allow Feature ID types to be interleaved. It allows arrays like this:
`[{FeatureIdAttribute}, {FeatureIdAttribute}]`
OR this:
`[{FeatureIdImplicitRange}, {FeatureIdImplicitRange}]`
but the old code also allowed this:
`[{FeatureIdAttribute}, {FeatureIdImplicitRange}]`.
Can you update the syntax here to allow interleaved types? 

I think there may be a couple other places like this one, where we want to allow arrays with elements of different types.

Reply:
Well, this is my careless. I will roll back to `Array<ModelComponents.FeatureIdAttribute|ModelComponents.FeatureIdImplicitRange|ModelComponents.FeatureIdTexture>`.